user-interface/src/main/java/life/qbic/datamanager/views/projects/create/AddProjectDialog.java

```diff
@@ -77,8 +74,8 @@ public AddProjectDialog(ProjectInformationService projectInformationService,
       SpeciesLookupService speciesLookupService,
       ContactRepository contactRepository, TerminologyService terminologyService) {
     super();
-
-    initCancelShortcuts();
+    requireCloseConfirmation();
+    setIgnoreCloseCheckIfUnmodified(false);
```

Comment:
this method sounds like a good idea, but why is it set to false?

Reply:
it is set to false as we did not implement the modification tracking. So as to not change the behaviour it is set to false right now. I will add a comment.